examples/openstack_example.rb

```diff
@@ -0,0 +1,61 @@
+require sprintf('%s/../%s', File.dirname(File.expand_path(__FILE__)), 'path_helper')
+
+require 'rouster'
+require 'plugins/openstack' # brings in fog and some helpers
+
+ostack = Rouster.new(
+  :name      => 'ostack-testing',
+  :sudo      => false,
+  :logfile   => true,
+  :passthrough => {
```

Comment:
ah, that was ambiguous. i meant can you true up the spacing in front of the comments, or just remove the spacing altogether - i think some of the comments were valuable